doc/man/8/cephadm.rst

```diff
@@ -64,14 +64,17 @@ Synopsis
 |                           [--dashboard-crt DASHBOARD_CRT]
 |                           [--ssh-config SSH_CONFIG]
 |                           [--ssh-private-key SSH_PRIVATE_KEY]
-|                           [--ssh-public-key SSH_PUBLIC_KEY] 
+|                           [--ssh-public-key SSH_PUBLIC_KEY]
 |                           [--ssh-user SSH_USER] [--skip-mon-network]
 |                           [--skip-dashboard] [--dashboard-password-noupdate]
 |                           [--no-minimize-config] [--skip-ping-check]
 |                           [--skip-pull] [--skip-firewalld] [--allow-overwrite]
 |                           [--allow-fqdn-hostname] [--skip-prepare-host]
 |                           [--orphan-initial-daemons] [--skip-monitoring-stack]
 |                           [--apply-spec APPLY_SPEC]
+|                           [--registry-url REGISTRY_URL]
+|                           [--registry-username REGISTRY_USERNAME]
+|                           [--registry-password REGISTRY_PASSWORD]
```

Comment:
I wonder if sending passwords as command options is a security risk. Note that we are already doing it on `--initial-dashboard-password`, but by default, the dashboard user will be forced to change the password on the first login.

Maybe allowing the user to choose between `--registry-password` or `--registry-password-stdin`/`--registry-password-file`[1][2] fixes the issue for users that are more concerned about security, but I would prefer to avoid adding any unnecessary complexity, so if this is how `cephadm` deals with these scenarios, we don't have to change it.

@sebastian-philipp @adk3798 others, any thoughts about this?

[1] https://docs.docker.com/engine/reference/commandline/login/
[2] https://www.mankier.com/1/podman-login

Reply:
:+1: it is a security risk. 

maybe extend this to the existing `--config-json` arg?